azure/components/apigateway/id.ftl

```diff
@@ -1,13 +1,46 @@
 [#ftl]
+[#include "openapi.ftl"]
 
 [@addResourceGroupInformation
     type=APIGATEWAY_COMPONENT_TYPE
-    attributes=[]
+    attributes=[
+        {
+            "Names" : "ContentType",
+            "Description" : "The content type of the API's schema",
+            "Type" : STRING_TYPE,
+            "Default" : "application/vnd.oai.openapi.components+json"
+        },
+        {
+            "Names": "Contact",
+            "Description" : "The contact for the API Management Service",
+            "Children" : [
+                {
+                    "Names" : "Name",
+                    "Type": STRING_TYPE
+                },
+                {
+                    "Names" : "Email",
+                    "Type" : STRING_TYPE
+                }
+            ]
+        },
+        {
+            "Names" : "FrontDoor",
+            "Children": [
+                {
+                    "Names" : "Mapping",
```

Comment:
The mapping configuration here should align with the your comments about path, If enabled the Mapping should hide the base path. Thats what this configuration is used for over in AWS

Reply:
Removed this as after a refactor I found the existing Mapping attribute was sufficient. Refactor took into consideration your above comments so mapping blanks out the base path.